### ISSUE TITLE
Persepolis Keeps Breaking ISOs

### ISSUE THREAD

**Issue by NiffirgkcaJ** [persepolisdm/persepolis#1030]
**System Details:**
* Operating System: Linux
* Distro (For GNU/Linux and BSD users): CachyOS
* Desktop Environment (For GNU/Linux and BSD users): GNOME 47
* Persepolis Version: 5.0.1
* How do you install Persepolis? AUR

**Issue Description and steps to reproduce:**
Just like what I said on the title, the downloader keeps on breaking ISOs when it was downloaded successfully, even if those ISOs didn't get interrupted. This issue didn't happen when Persepolis still used Aria as its backend.

**Comment by alireza-amirsamimi** [comment]
Hello
Thanks for reporting this issue. Is the quality of your Internet good?

**Comment by NiffirgkcaJ** [comment]
Yes, my internet is good. The speed at which it is downloading is quite fast, and it doesn't get interrupted, but when I try to “burn” ISOs downloaded with Persepolis now, it always breaks as opposed to ISOs downloaded normally through the browser downloader.

**Comment by cbdejavu** [comment]
This is probably a duplicate of #1033 
Maybe the title of #1033 should be changed to "Large files are being corrupted on 5.0 series of persepolis".  As mentioned in that other thread the only way I could successfully get large files to download and validate their checksum's properly was to rollback to the 4.2.0 version.  This is something that was definitely introduced in the 5.0 version of the product.  I'll spin up a clean Windows 11 VM and test the windows version to see if windows is also corrupting the files tomorrow when I get a chance.  Basically for testing I'm just downloading Linux Distro ISOs that are in the 4gb size range.  Since most if not all linux distributions have some sort of checksum available for their ISOs it makes an excellent way to test if the file is being corrupted during the download process.

**Comment by NiffirgkcaJ** [comment]
I don't see this as a duplicate of that since this was reported before it, but the issue seems to be the same.

During the 4.2.0 version, Persepolis used Aria, but in the 5.0.0 version, it began using its own download manager backend. I think this change is the reason for the corruption, but I haven't confirmed it yet.

**Comment by Sadi58** [comment]
I have the same problem of md5sum failure (Debian Linux and KDE Plasma 5) and hope installing version 4.2.0 will save me from this until the problem is resolved.
